libs/fluid-elements/popover/jest.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
this file can be removed and changed to `jest.config.json`

```json
{"name": "fluid-popover"}
```
That is needed to test with bazel